contracts/proxy-lib/tests/common/counter_helper.rs

```diff
@@ -0,0 +1,34 @@
+use eyre::Result;
+use near_workspaces::{network::Sandbox, Contract, Worker};
+
+use super::deploy_contract;
+
+const COUNTER_WASM: &str = "../test-counter/res/test_counter_near.wasm";
+
+#[derive(Clone)]
+pub struct CounterContracttHelper {
+    pub counter_contract: Contract,
+}
+
+impl CounterContracttHelper {
```

Comment:
CounterContractHelper
```suggestion
impl CounterContracttHelper {
```